uktrade/data-hub-frontend
# Support the v4 contacts endpoint

## Description of change

Allows contacts endpoint to be switched from v3 to v4 by enabling the `address-area-contact-required-field` feature flag. This feature flag should not be turned on yet until the area field is added to contacts.  
This also adds testing around some functionality that made use of contacts.  
This PR will be squash merged to eliminate the WIP commits.

## Test instructions

When `address-area-contact-required-field` is enabled the contacts form will fail to submit, in the network response you should see a validation error.

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
